### PR TITLE
4800 reports docs changes

### LINKED ISSUE #88: Update documentation for simplified report modal
Update docs to match new report modal look:

Screenshot 2024-09-17 at 2 46 55 PM

Wording has been changed and non default fields are removed.
Also search functionality for code and name (and codeOrName) fields will be startsWith rather than Like.

See omSupply PR for details: https://github.com/msupply-foundation/open-msupply-reports/pull/93

### PR DESCRIPTION
**Pull request by fergie-nz**
Closes #88

Adds user facing docs for standard reports

### CODE REVIEW

File: content/docs/getting_started/report.md [hunk 1]
```diff
@@ -82,4 +82,14 @@ The report will be downloaded as an excel file.
 
 Standard reports will be translated to the user's language if available when generating the report.
 
-English will be used by default in places where translations in the user's language is unavailable.
\ No newline at end of file
+English will be used by default in places where translations in the user's language is unavailable.
+
+## Standard Reports
+
+Builds 2.3.0 and above will include standard reports by default in all instances of open mSupply. The 2.3.0 build includes the following reports:
```
Comment (roxy-dao) [inline content/docs/getting_started/report.md:89]:
```suggestion
Version 2.3.0+ will include standard reports by default in all instances of Open mSupply:
```

Are these the default reports that will be available?

**Review by roxy-dao — CHANGES_REQUESTED** [review]
Can you also update the screenshots for the report args and so on as well if needed? Thanks Ferg

File: content/docs/getting_started/report.md (hunk 1 above)
Comment (fergie-nz) [reply content/docs/getting_started/report.md:89]: Yes default reports available.

They're part of the OMS repo now

**Review by roxy-dao — APPROVED** [review]
Thanks Ferg!

**Pull request merged**